docs/README.md

```diff
@@ -1,20 +1,56 @@
 ## Generate docs automatically
 
-Run
+To generate documentation for `Games.jl`, run
+
 ```
 $julia make.jl
 ```
 
-The generated docs can be accessed in `build/index.html`.
+The generated docs can be accessed in `docs/build/index.html`.
+
+It includes three parts to generate the documentation.
+
+### auto_doc_gen.jl
+
+Main part of documentation generation. It reads `src/Games.jl` to find
+out all the source files, and then write markdown files for each with
+the structure instructed by `docs/build/Structure`, which will be utilized
+by "Documenter.jl" later.
+
+Especially, pattern of each page could be modified by changing the
+strings in `auto_doc_gen.jl`, called `file_page` and `section_page`.
+
+### manually written files
 
-You may modify `Structure` to arrange the structure of the website to be generated. For example:
+Two files should be written manually:
```

Comment:
I see, I think "should" is a bit unclear here about whether you can or must do it. Instead, I would say:
"While not necessary, it is (strongly?) recommended to manually specify the two following files:"